repository.tf

```diff
@@ -82,21 +76,34 @@ resource "github_repository_ruleset" "all" {
     }
   }
 
-  bypass_actors {
-    actor_id    = 15368 # Github Actions bot app ID was found here: https://api.github.com/apps/github-actions
-    actor_type  = "Integration"
-    bypass_mode = "pull_request"
-  }
-
   rules {
     creation            = false # do not restrict creation 
     update              = false
     deletion            = false
-    required_signatures = true
     non_fast_forward    = false
   }
 }
 
+resource "github_repository_ruleset" "required_signatures" {
```

Comment:
Why create a new resource?
I guess what you want to do is change `github_repository_ruleset.all.conditions.ref_name.exclude` from hardcoded value to `each.value.exclude_rules.required_signatures.branches`